runtime/src/main/java/org/corfudb/runtime/ServerTriggeredCheckpointer.java

```diff
@@ -57,7 +57,7 @@ private CheckpointWriter<ICorfuTable<?,?>> getCheckpointWriter(TableName tableNa
         CorfuTable<CorfuDynamicKey, OpaqueCorfuDynamicRecord> corfuTable = openTable(tableName, keyDynamicProtobufSerializer,
                 checkpointerBuilder.cpRuntime.get());
         CheckpointWriter<ICorfuTable<?,?>> cpw =
-                new CheckpointWriter(checkpointerBuilder.corfuRuntime, streamId, "ServerCheckpointer", corfuTable);
+                new CheckpointWriter(checkpointerBuilder.cpRuntime.get(), streamId, "ServerCheckpointer", corfuTable);
```

Comment:
Why are there two runtime instances?

Reply:
We need 2 runtimes since some tables are opened using the regular ProtobufSerializer and other using KeyDynamicProtobufSerializer

Reply:
Which tables, metadata tables and user tables?

Reply:
Also, shouldn't `CorfuStoreCompactorConfig` disable caching?

Reply:
Yes, metadata and user tables
About disabling cache, yes it should. Will update that as well.